src/main/java/io/openliberty/tools/common/plugins/util/ServerFeatureUtil.java

```diff
@@ -380,18 +382,39 @@ private Properties getBootstrapProperties(File bootstrapProperties) {
     private String evaluateExpression(Properties properties, String expression) {
         String value = expression;
         if (expression != null) {
-            Pattern p = Pattern.compile("\\$\\{([^\\}]*)\\}");
+            Pattern p = Pattern.compile("\\$\\{(.*?)\\}");
```

Comment:
I think this will cause a problem if the expression passed in contains two variables like the following:

${var1}/${var2}

That initial pattern was supposed to do a non greedy match I think and find the first variable, which would return `var1` at line 389. After your change to the pattern, I think it will return `var1/${var2`.

Reply:
The change is fine. I see that the ? makes it non-greedy.